src/test/java/com/flightstats/hub/dao/aws/ClusterContentServiceTest.java

```diff
@@ -0,0 +1,96 @@
+package com.flightstats.hub.dao.aws;
+
+import com.flightstats.hub.cluster.LastContentPath;
+import com.flightstats.hub.dao.ChannelService;
+import com.flightstats.hub.dao.ContentDao;
+import com.flightstats.hub.model.*;
+import com.flightstats.hub.util.HubUtils;
+import com.flightstats.hub.util.TimeUtil;
+import lombok.SneakyThrows;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClusterContentServiceTest {
+    @Mock private ClusterContentService ccs;
+    @Mock private ChannelService channelSvc;
+    @Mock private ContentDao mockSpokeWriteDao, mockSpokeReadDao;
```

Comment:
I don't know how everybody else feels about multiple member declarations on one line. I think I prefer separate lines even if the ones you've grouped are related; it seems more consistent and therefore less surprising. Others should weigh in here, though, since we don't have an explicit convention related to this.

Reply:
Yeah, I usually prefer individual lines too.  In this case it felt like it would help clarify similarities amongst a bunch of similarly-typed vars.  I'm not invested though; whichever you all prefer works for me.

Reply:
If you'd usually leave it as two, being consistent probably trumps the grouping, imo. The names are similar enough that it shouldn't be too bad to associate them. 🤷‍♂️

Reply:
s/this case/cases like this, probably for me...but not everyone might agree with the grouping...it might be more aligned with how I think through this class.  🤷‍♂️ as well.  I'll split them out.

Reply:
The convention in the Portland office is to have one per line, though I'm not sure it's written down anywhere. And I think it's generally looked down upon in the Java community.